src/common/options/global.yaml.in

```diff
@@ -1262,6 +1262,12 @@ options:
   desc: Inject various internal delays to induce races (seconds)
   default: 0
   with_legacy: true
+- name: ms_inject_network_block
+  type: float
```

Comment:
how about:

1. rename it to `ms_inject_network_congestion`
1. use `uint`, to be consistent with `ms_inject_socket_failures`. as in this change `ms_inject_network_block` is actually a "ratio", while `ms_inject_socket_failures` is a period.

Reply:
Hi, @tchaikov 
Sorry for the late reply, I am not sure of the above comment.

Do you mean that rename the config name and its type?
Then try to make it to be a ratio when  `ms_inject_socket_failures` is set up?

Reply:
I rename this to `ms_inject_network_congestion`
That more like `ms_inject_socket_failures` and `ms_inject_network_congestion` means stuck with N times.